### ISSUE TITLE
Tried to define vaadin-iconset version undefined when version 24.4.0-alpha22 is already in use.

### ISSUE THREAD

**Issue by javier-godoy** [FlowingCode/FontAwesomeIronIconset#105]
I'm trying to upgrade the demo to 24.4.0. When the application starts, the following message is logged:
![image](https://github.com/FlowingCode/FontAwesomeIronIconset/assets/11554739/1bdb6eeb-0a86-47a6-9051-030aa4065b59)

(It works in 24.3, though)

**Comment by javier-godoy** [comment]
After upgrading Vaadin to 24.0.0.beta2 in pom:
```
>npm why node_modules/@flowingcode/font-awesome-iron-iconset/node_modules/@vaadin/icon
@vaadin/icon@24.3.13
node_modules/@flowingcode/font-awesome-iron-iconset/node_modules/@vaadin/icon
  @vaadin/icon@"^24.0.0" from @flowingcode/font-awesome-iron-iconset@5.1.1
  node_modules/@flowingcode/font-awesome-iron-iconset
    @flowingcode/font-awesome-iron-iconset@"5.1.1" from the root project
```

**Comment by javier-godoy** [comment]
The client-side code has dependencies with Vaadin `^24.0.0`, which ignores pre-release qualifiers (thus it resolved to `24.3.`something by the time this ticket was created).

https://github.com/FlowingCode/FontAwesomeIronIconset/blob/42b8d6db5173a05b2a3ed9219b1a7cc7d04f63f3/src/main/javascript/font-awesome-iron-iconset/package.json#L22-L25

Now that 24.4 has been released, the error is gone. 

Closing since it works as intended.
